Podfile

```diff
@@ -1,3 +1,4 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
-pod 'React', :git => 'https://github.com/facebook/react-native'
+pod 'React'
```

Comment:
I actually originally left it there, but it was yelling at me. When I simplified it down to just React it worked :confused: ... end of the day seemed like working was an improvement